packages/consumption/test/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.test.ts

```diff
@@ -973,15 +1061,14 @@ describe("ReadAttributeRequestItemProcessor", function () {
 
             test("can be called with an arbitrary third party if the thirdParty string array of the ThirdPartyRelationshipAttributeQuery contains an empty string", async function () {
                 const sender = CoreAddress.from("Sender");
-                const aThirdParty = CoreAddress.from("AThirdParty");
-                const anUninvolvedThirdParty = CoreAddress.from("AnUninvolvedThirdParty");
+                const anInvolvedThirdParty = CoreAddress.from("AnInvolvedThirdParty");
```

Comment:
Could be my fault fornot knowing what "involved" means in this context, but I was fairly confused by this naming where the contrast between `aThirdParty`and `anUninvolvedThirdParty` was lost

Reply:
I can understand that, that was also my concern. What do you think about `aQueriedThirdParty` instead of `anInvolvedThirdParty` as an alternative naming?

Reply:
I like that